Logging/ARKDataArchive.h

```diff
@@ -40,13 +37,13 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, readonly) NSUInteger trimmedObjectCount;
 
 /// The URL of the archive file.
-@property (nonatomic, copy, readonly) NSURL *archiveFileURL;
+@property (nonnull, nonatomic, copy, readonly) NSURL *archiveFileURL;
 
 /// Archives the provided object (on the calling thread), and queues appending it to the archive.
-- (void)appendArchiveOfObject:(id <NSSecureCoding>)object;
+- (void)appendArchiveOfObject:(nonnull id <NSSecureCoding>)object;
 
 /// Reads in all contents of the archive, unarchives each object, and returns them on the main thread.
-- (void)readObjectsFromArchiveWithCompletionHandler:(nonnull void (^)(NSArray *unarchivedObjects))completionHandler;
+- (void)readObjectsFromArchiveWithCompletionHandler:(nonnull void (^)(NSArray * __nonnull unarchivedObjects))completionHandler;
```

Comment:
Isn't the new way to use [`_Nonnull`](https://developer.apple.com/swift/blog/?id=25)?